src/media/local-stream.ts

```diff
@@ -88,14 +88,17 @@ abstract class _LocalStream extends Stream {
   /**
    * Set the mute state of this stream.
    *
+   * Note: This sets the user-toggled mute state, equivalent to changing the "enabled" state of the
+   * track. It is separate from the browser-toggled mute state.
+   *
    * @param isMuted - True to mute, false to unmute.
    */
   setMuted(isMuted: boolean): void {
     if (this.inputTrack.enabled === isMuted) {
       this.inputTrack.enabled = !isMuted;
       // setting `enabled` will not automatically emit MuteStateChange, so we emit it here
       if (!this.inputTrack.muted) {
-        this[StreamEventNames.MuteStateChange].emit(isMuted);
+        this[StreamEventNames.MuteStateChange].emit(isMuted, MuteStateChangeReason.ByUser);
```

Comment:
I would also like to have reason added as an argument to setMuted() (with default value being MuteStateChangeReason.ByUser) so that we can use it for other things like remote mute - see https://github.com/webex/webex-js-sdk/blob/0692849ddb041ee642b9c2b6c589a54e7a242637/packages/%40webex/media-helpers/src/webrtc-core.ts#L29 for the current list of reasons used in SDK that we need on top of "by user" and "by browser"